api/registration/models.py

```diff
@@ -77,4 +85,52 @@ def password_reset_token_created(sender, instance, reset_password_token, *args,
     subject = 'Reset your password'
     message = f'Dear {reset_password_token.user.full_name},\n\nClick on the link below to reset your password.\n{absolute_url}\n\nCLIMAN Team\n2023'
     msg = EmailMessage(subject, message, to=[reset_password_token.user.email])
-    msg.send()
\ No newline at end of file
+    msg.send()
+
+
+class Appointments(models.Model):
+    # The primary key for the Appointments model
+    # is a UUID field that is generated automatically.
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
Replace the uuidfield with BigAutoField
With primary key option

Reply:
ok will get on that immediately